package.json

```diff
@@ -20,7 +20,7 @@
     "README"
   ],
   "dependencies": {
-    "@sentry/node": "^6.2.0",
+    "@sentry/node": "7.0.0-beta.2",
     "@sentry/tracing": "^6.11.0",
```

Comment:
I would suggest bumping `@sentry/tracing` to `7.0.0-beta.2`, too. We generally encourage to keep all Sentry JS packages up to date at the same version because they rely on each other. Also, we did change some things in the Tracing package (especially in `beta.2` with the new `baggage` header) so it would be great to test these, too. 
(see Changelog and Migration guide for more info)